zawata/S19-Team2
# Us54 t76 to j2000

To test:
1.  In config, create config.json with the following contents:
`
{
  "main_subject": -39,
  "subject_name": "LMAP",
  "kernels": [
    "LMAP_FullTrajectory.bsp",
    "latest_leapseconds.tls.pc"
  ]
}
`
2. In config/kernels, add LMAP_FullTrajectory.bsp from the team drive, and latest_leapseconds.tls.pc from slack.
3. Build and run
4. Make an http GET request to the /api/toUTC endpoint with a number argument. http://localhost:5000/api/toUTC?time=-60000000000 should give you
`{
"J2000": -60000000000,
"UTC": "98-09-04T01:19:19"
}`
5. Make an http get request to the /api/toJ2000 endpoint with an ISO 8601 string argument. http://localhost:5000/api/toJ2000?time=2012-12-25T3:00:00 should give you
`{
"J2000": 409676467.18373287,
"UTC": "2012-12-25T3:00:00"
}`
